test/web-platform-tests/to-upstream/svg/element-svg.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<title>SVGSVGElement interface</title>
+<link rel="author" title="Timothy Gu" href="mailto:timothygu99@gmail.com">
+<link rel="help" href="https://svgwg.org/svg2-draft/struct.html#InterfaceSVGSVGElement">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<svg>
+  <g id="group"></g>
+</svg>
+<svg id="second" requiredExtensions=" abc  def "></svg>
+
+<script>
+"use strict";
+
+test(() => {
+  const svg = document.querySelector("svg");
+  assert_equals(svg.namespaceURI, "http://www.w3.org/2000/svg");
+  assert_equals(svg.localName, "svg");
+  assert_equals(svg.ownerSVGElement, null);
+  assert_equals(svg.viewportElement, null);
+  assert_true(svg instanceof Element);
+  assert_true(svg instanceof SVGElement);
+  assert_true(svg instanceof SVGGraphicsElement);
+  assert_true(svg instanceof SVGSVGElement);
+  assert_false(svg instanceof HTMLElement);
+}, "basic properties");
+
+test(() => {
+  const svg = document.querySelector("svg");
+  assert_equals(svg.getElementById("group"), svg.children[0]);
+  assert_equals(document.getElementById("group"), svg.children[0]);
+}, "getElementById()");
+
+test(() => {
+  const svg = document.createElementNS("http://www.w3.org/2000/svg", "svg");
+  assert_true(svg.requiredExtensions instanceof SVGStringList);
+  assert_true(svg.systemLanguage instanceof SVGStringList);
+  assert_array_equals(svg.requiredExtensions, []);
+  assert_array_equals(svg.systemLanguage, []);
+  svg.setAttribute("systemLanguage", "abc\r\ndef\f");
+  assert_array_equals(svg.requiredExtensions, []);
+  assert_array_equals(svg.systemLanguage, ["abc", "def"]);
```

Comment:
The [specification](https://svgwg.org/svg2-draft/struct.html#ConditionalProcessingSystemLanguageAttribute) says that the value of systemLanguage is a set of comma-separated tokens rather than space-separated tokens as with requiredExtensions, so this does not seem right.